bootloader/README.md

```diff
@@ -50,72 +54,67 @@ New firmware does not run properly but sets the valid bit and refreshes the watc
 # Using the bootloader
 
 ## Bootloader graphic
+
 The bootloader loads a graphic (Pinetime logo) from the SPI Flash memory. If this graphic is not loaded in the memory, the LCD will display garbage (the content of the SPI flash memory).
 
 The SPI Flash memory is not accessible via the SWD debugger. Use the firmware 'pinetime-graphics' to load the graphic into memory. All you have to do is build it and program it at address 0x00 :
 
- - Build:
+- Build:
+
 ```
 $ make pinetime-graphics
 ```
 
- - Program (using OpenOCD for example) :
+- Program (using OpenOCD for example) :
+
 ```
 program pinetime-graphics.bin 0
 ```
 
- - Let it run for ~10s (it does nothing for 5 seconds, then write the logo into the SPI memory, then (slowly) displays it on the LCD).
+- Let it run for ~10s (it does nothing for 5 seconds, then write the logo into the SPI memory, then (slowly) displays it on the LCD).
 
 ## Bootloader binary
+
 The binary comes from https://github.com/lupyuen/pinetime-rust-mynewt/releases/tag/v5.0.4
 
 It must be flash at address **0x00** in the internal flash memory.
 
 Using OpenOCD:
 
-`
-program bootloader-5.0.4.bin 0
-`
+`program bootloader-5.0.4.bin 0`
 
 ## Application firmware image
+
 Build the binary compatible with the booloader:
 
-`
-make pinetime-mcuboot-app
-`
+`make pinetime-mcuboot-app`
 
 The binary is located in *<build directory>/src/pinetime-mcuboot-app.bin*.
 
 It must me converted into a MCUBoot image using *imgtool.py* from [MCUBoot](https://github.com/mcu-tools/mcuboot/tree/master/scripts). Simply checkout the project and run the script <mcuboot root>/scripts/imgtool.py with the following command line:
 
-`
-imgtool.py  create --align 4 --version 1.0.0 --header-size 32 --slot-size 475136 --pad-header <build directory>/src/pinetime-mcuboot-app.bin image.bin
-`
+`imgtool.py create --align 4 --version 1.0.0 --header-size 32 --slot-size 475136 --pad-header <build directory>/src/pinetime-mcuboot-app.bin image.bin`
 
 The image must be then flashed at address **0x8000** in the internal flash memory.
 
 Using OpenOCD:
 
-`
-program image.bin 0x8000
-`
+`program image.bin 0x8000`
 
 ## OTA and DFU
+
 Pack the image into a .zip file for the NRF DFU protocol:
 
-`
-adafruit-nrfutil dfu genpkg --dev-type 0x0052 --application image.bin dfu.zip
-`
+`adafruit-nrfutil dfu genpkg --dev-type 0x0052 --application image.bin dfu.zip`
```

Comment:
I'd use a codeblock here with `sh` style to get syntax highlighting (can't do the suggestion properly because the \`\`\` are used for the codeblock and for the github suggestion)

```suggestion
```sh
adafruit-nrfutil dfu genpkg --dev-type 0x0052 --application image.bin dfu.zip
```
```